css/index.css

```diff
@@ -1,3 +1,375 @@
+}
+
+header nav ul, header nav ul li {
+	display: flex;
+	text-align: center;
+}
+
+header nav ul {
+  width: 70%;
+  align-self: center;
+  justify-content: space-between;
+	align-items: center
+}
+
+li a {
+	text-decoration: none;
+	color: #cccccc;
+	text-align: inherit;
+}
+
+li a:hover {
+	color: #000000;
+}
+
+.active {
+	color: #666666
+}
+
+nav > a {
+  display: flex;
+  align-self: center;
+	width: 20%;
+}
+
+.logo {
+	display: flex;
+	max-width: 100%;
+}
+
+header .header-container {
+	align-items: center;
+	justify-content: space-between;
+}
+
+header .header-container .item {
+  align-self: center;
+	display: flex;
+	flex-direction: column;
+	max-width: 48%;
+	width: 48%;
+	align-items: center;
+  justify-content: center;
+}
+
+header .header-container .item h1 {
+	text-align: center;
+	font-size: 3.4rem;
+}
+
+header .header-container .item button {
+	margin: 0 25% 5rem 25%;
+	width: 50%;
+	border: 1px solid black;
+	border-radius: 0;
+	padding: 1rem 0;
+	font-size: 1.6rem;
+  cursor: pointer;
+  background-color: white;
+}
+
+header section .text-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+header .header-container .header-img {
+	background: url('../img/header-img.png') no-repeat;
+	width: 48%;
+  height: 40rem;
+}
+
+/* Main portion styling */
+main section .upper-text-container {
+	width: 49%;
+}
+
+.banner {
+	display: block;
+  width: 98%;
+  padding: 0 1%;
+}
+
+main section .lower-text-container {
+	width: 32%;
+}
+
+/* Services main portion specific styling */
+main .item-container .service-item {
+	display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: stretch;
+  width: 49%;
+	background-color: #eeeeee;
+	border: 1px solid black;
+	padding: 10px;
+}
+
+main .item-container .service-item .service-btn {
+	margin: 20px 0;
+  width: 40%;
+  background-color: #fff;
+	border: 1px solid black;
+	border-radius: 4px;
+	padding: 10px 0;
+	font-size: 100%;
+	cursor: pointer;
+}
+
+/* Footer portion styling */
+footer {
+	margin: 4rem 0 2rem 1rem;
+}
+
+.copyright {
+	display: block;
+	margin: 10rem 0;
+	text-align: center;
+}
+
+/* Services footer portion specific styling */
+footer section .text-container {
+  width: 49%;
+}
+.bullet {
+	margin-top: 20px;
+	list-style-type: square;
+  list-style-position: inside;
+  text-align: left;
+}
+
+.bullet-item {
+	margin-right: 0;
+  padding: 7px 0;
+  text-align: left;
+}
+
+/* Responsive design using media queries */
+@media screen and (max-width: 800px){
+  .container {
+    padding: 0 2%;
+  }
+  header nav {
+    flex-direction: column-reverse;
+  }
+  header nav ul {
+    width: 96%;
+    padding: 0 2%;
+  }
+  header nav ul li {
+    margin: 0;
+  }
+  nav > a {
+    width: 25%;
+    margin-bottom: 3em;
+  }
+  header .header-container {
+    justify-content: space-around;
+  }
+  header .header-container .header-img {
+    display: none;
+  }
+  main section .lower-text-container {
+    width: 31%;
+  }
+}
+
```

Comment:
You might want to add more comments to your css :)

Reply:
Noted ✅